buildtest/cli/__init__.py

```diff
@@ -82,6 +83,35 @@ def positive_number(value):
     return int_val
 
 
+def supported_color(value):
+    """Checks if input is positive number and returns value as an int type.
```

Comment:
please change this docstring you must have copied this from another method

Reply:
changed